content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -363,10 +363,10 @@ null `namespaceSelector` matches the namespace of the Pod where the rule is defi
 {{< feature-state for_k8s_version="v1.29" state="alpha" >}}
```

Comment:
This feature-state line also needs an update to show it's in the 'beta' stage for version 1.30. It'd be great if we could use `{{< feature-state feature_gate_name="MatchLabelKeysInPodAffinity >}}` but the improved shortcut functionality isn't synced with `dev-1.30` yet, a manual bump might be necessary for now.

```suggestion
{{< feature-state for_k8s_version="v1.30" state="beta" >}}
```